php-frontend/src/main/java/org/sonar/php/parser/TreeFactory.java

```diff
@@ -308,11 +308,15 @@ public VariableDeclarationTree constDeclaration(InternalSyntaxToken identifierTo
   }
 
   public UseClauseTree useClause(NamespaceNameTree namespaceName, Optional<Tuple<InternalSyntaxToken, InternalSyntaxToken>> alias) {
+    return groupUseClause(Optional.<InternalSyntaxToken>absent(), namespaceName, alias);
+  }
+
+  public UseClauseTree groupUseClause(Optional<InternalSyntaxToken> useTypeToken, NamespaceNameTree namespaceName, Optional<Tuple<InternalSyntaxToken, InternalSyntaxToken>> alias) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 181 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS00103)